src/SwedbankPay.Sdk/Payments/MobilePayPayments/MobilePayPaymentResponse.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Globalization;
+
+using SwedbankPay.Sdk.PaymentOrders;
+
+namespace SwedbankPay.Sdk.Payments.MobilePayPayments
+{
+    public class MobilePayPaymentResponse
+    {
+        public MobilePayPaymentResponse(PaymentResponseObject payment,
+                               OperationList operations)
+        {
+            this.Payment = payment;
+            this.Operations = operations;
+        }
+
+        public PaymentResponseObject Payment { get; }
+        public OperationList Operations { get; }
+    }
+
+    public class PaymentResponseObject
+    {
+        public PaymentResponseObject(Uri id,
+                               string number,
+                               PaymentInstrument instrument,
+                               DateTime created,
+                               DateTime updated,
+                               State state,
+                               Operation operation,
+                               Intent intent,
+                               CurrencyCode currency,
+                               Amount amount,
+                               string description,
+                               string payerReference,
+                               string initiatingSystemUserAgent,
+                               string userAgent,
+                               CultureInfo language,
+                               PricesListResponse prices,
+                               TransactionListResponse transactions,
+                               MobilePayPaymentAuthorizationListResponse authorizations,
+                               CapturesListResponse captures,
+                               ReversalsListResponse reversals,
+                               CancellationsListResponse cancellations,
+                               Urls urls,
+                               PayeeInfo payeeInfo,
+                               MetaDataResponse metaData)
+        {
+            Id = id;
+            Number = number;
+            Created = created;
+            Updated = updated;
+            Instrument = instrument;
+            Operation = operation;
+            Intent = intent;
+            State = state;
+            Currency = currency;
+            Prices = prices;
+            Amount = amount;
+            Description = description;
+            PayerReference = payerReference;
+            InitiatingSystemUserAgent = initiatingSystemUserAgent;
+            UserAgent = userAgent;
+            Language = language;
+            Urls = urls;
+            PayeeInfo = payeeInfo;
+            Transactions = transactions;
+            Authorizations = authorizations;
+            Captures = captures;
+            Reversals = reversals;
+            Cancellations = cancellations;
+            MetaData = metaData;
```

Comment:
`metadata` is written with a lowercase `d` in the API, so please make that consistent throughout the SDK as well. Perhaps not in this PR, but in a separate one.

```suggestion
            Metadata = metadata;
```

Reply:
It seems like uppercase `d` is used in the SDK. I'll make a seperate Jira-task for this.